dataactcore/models/errorModels.py

```diff
@@ -46,4 +46,7 @@ class ErrorMetadata(Base):
     occurrences = Column(Integer)
     first_row = Column(Integer)
     rule_failed = Column(Text, nullable=True)
+    file_type_id = Column(Integer)
```

Comment:
I'm curious about the original reasoning for having the error tables in a different table than the validation tables, since those two functions are so closely related. Now that we're referencing file types here, we might stand to gain from combining these databases to take advantage of foreign keys and sqlalchemy relationships.